patches/src/main/kotlin/app/revanced/patches/spotify/layout/theme/CustomThemePatch.kt

```diff
@@ -125,6 +126,38 @@ private val customThemeBytecodePatch = bytecodePatch {
             SETTINGS_HEADER_COLOR_LITERAL,
             backgroundColorSecondary!!
         )
+
+        // Hijacks a util method that removes alpha to replace hardcoded accent colors
```

Comment:
Comments end with punctuation
```suggestion
        // Hijacks a util method that removes alpha to replace hardcoded accent colors.
```